src/py/flwr/server/app.py

```diff
@@ -76,7 +77,8 @@ def start_server(
         initialized_config["num_rounds"],
     )
 
-    _fl(server=initialized_server, config=initialized_config)
+    _fl(server=initialized_server, config=initialized_config,
+        force_final_distributed_eval=force_final_distributed_eval)
```

Comment:
This will fail the formatting check - you can fix it by running `./dev/format.sh` (make sure to have the dev dependencies installed beforehand)